.travis.yml

```diff
@@ -37,16 +34,18 @@ before_install:
     - pip install -U pip pipenv wheel
     - pipenv install --dev --deploy
     - pipenv check
+    - pushd /opt && if [ ! -d jsonnet ]; then git clone --branch v0.12.1 --depth 1 https://github.com/google/jsonnet.git; fi && cd jsonnet && make && popd
 cache:
     - yarn
     - pip
     - directories:
         - /opt/python/
+        - /opt/jsonnet/
         - /home/travis/.local/share/
 script:
     - set -e
     - yarn compile
```

Comment:
We could remove this since it would be run in `scripts/build.sh` if the `static` folder doesn't exist?